extmod/modussl_mbedtls.c

```diff
@@ -74,6 +75,24 @@ STATIC void mbedtls_debug(void *ctx, int level, const char *file, int line, cons
 }
 #endif
 
+STATIC NORETURN void mbedtls_raise_error(int err) {
+    if (err < 0 && err > -120) {
+        // err comes from underlying sockets, not mbedtls
+        mp_raise_OSError(-err);
+    }
+    #if 1
+    // Including mbedtls_strerror takes about 16KB on the esp32 due to all the strings
+    char error_buf[80];
```

Comment:
Is that guaranteed to be enough, and if so: how?

Reply:
Eyeballing the error string file it should be enough. If it's not, mbedtls_strerror truncates the error message correctly (i.e., not buffer overrun). I don't think there's a "max_length" constant I could pull in.